c/tskit/tables.c

```diff
@@ -7732,25 +7732,36 @@ tsk_table_collection_free(tsk_table_collection_t *self)
  * tables. We do not consider the file_uuids either, since this is a property of
  * the file that set of tables is stored in. */
 bool
-tsk_table_collection_equals(tsk_table_collection_t *self, tsk_table_collection_t *other)
+tsk_table_collection_equivalence(
+    tsk_table_collection_t *self, tsk_table_collection_t *other, tsk_flags_t options)
```

Comment:
I think to match `table_collection_equals` this should be `table_collection_equivalent`?

Reply:
agree!

Reply:
was waiting to do that at the end at once